quickstarts/trusted-artifact-signer.yaml

```diff
@@ -0,0 +1,59 @@
+  introduction: 'Red Hat Trusted Artifact Signer enables you to sign and verify container images and other build artifacts to ensure integrity and provenance. In this quick start, you will install the operator, create an instance, and sign your first artifact.'
+  prerequisites:
+    - Administrator access to the OpenShift cluster
+    - Access to Operators → OperatorHub
+  tasks:
+    - description: |-
+        Trusted Artifact Signer brings verifiable integrity and provenance to your software supply chain by signing container images and other build artifacts. Use signatures and attestations to enable admission policies and establish trusted builds.
+
+        Highlights:
+        - Sign images and generate attestations with Sigstore/cosign
+        - Enforce signature policies at admission
+        - Integrates with Trusted Profile Analyzer (SBOM analysis) and Advanced Cluster Security (runtime)
+
+        Watch next:
+        - [Trusted Artifact Signer overview](https://www.youtube.com/watch?v=C3I7DTBmTg8)
+        - [Trusted Software Supply Chain overview](https://www.youtube.com/watch?v=xRmBjklNXhc)
+        - [Trusted Software Supply Chain interactive demo](https://app.arcade.software/share/nkfw8tMy4MNE6zxiET31?ref=share-link)
+      title: Why Trusted Artifact Signer?
+    - description: |-
+        Go to Operators → OperatorHub and search for "Trusted Artifact Signer". Open the tile and click Install. After installation, verify the operator shows Succeeded under Operators → Installed Operators. 
+
```

Comment:
Is a line break missing here?
<img width="437" height="267" alt="image" src="https://github.com/user-attachments/assets/f203ba50-53e7-46ac-bbfa-407e960fff9c" />